<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U1OL4JI" failures="4" tests="4" name="com.triotree.test.website.FrontOffice.MarkPatientDeceasedTest" time="0.016" errors="0" timestamp="10 May 2021 08:02:45 GMT">
  <testcase classname="com.triotree.test.website.FrontOffice.MarkPatientDeceasedTest" name="markPatientDeceasedSearchPatientTest" time="0.016">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
]]>
    </failure>
  </testcase> <!-- markPatientDeceasedSearchPatientTest -->
  <testcase classname="com.triotree.test.website.FrontOffice.MarkPatientDeceasedTest" name="revokeDeceasedPatientTest" time="0.000">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
]]>
    </failure>
  </testcase> <!-- revokeDeceasedPatientTest -->
  <testcase classname="com.triotree.test.website.FrontOffice.MarkPatientDeceasedTest" name="frontOfficeCancellationOfAllAppointmentOnMarkPatientDeceased" time="0.000">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
]]>
    </failure>
  </testcase> <!-- frontOfficeCancellationOfAllAppointmentOnMarkPatientDeceased -->
  <testcase classname="com.triotree.test.website.FrontOffice.MarkPatientDeceasedTest" name="markPatientDeceasedTest" time="0.000">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at java.base/java.util.regex.Matcher.getTextLength(Matcher.java:1769)
	at java.base/java.util.regex.Matcher.reset(Matcher.java:416)
	at java.base/java.util.regex.Matcher.<init>(Matcher.java:253)
	at java.base/java.util.regex.Pattern.matcher(Pattern.java:1130)
	at com.aventstack.extentreports.utils.ExceptionUtil.getExceptionHeadline(ExceptionUtil.java:25)
	at com.aventstack.extentreports.model.ExceptionInfo.<init>(ExceptionInfo.java:19)
	at com.aventstack.extentreports.ExtentTest.log(ExtentTest.java:405)
	at com.aventstack.extentreports.ExtentTest.skip(ExtentTest.java:947)
	at com.aventstack.extentreports.ExtentTest.skip(ExtentTest.java:959)
	at com.triotree.test.base.ResultListener.onTestSkipped(ResultListener.java:71)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:778)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- markPatientDeceasedTest -->
</testsuite> <!-- com.triotree.test.website.FrontOffice.MarkPatientDeceasedTest -->
